cloudinit/sources/DataSourceOracle.py

```diff
@@ -332,6 +332,20 @@ def _add_network_config_from_opc_imds(self, set_primary: bool = False):
                 self._network_config["ethernets"][name] = interface_config
 
 
+class DataSourceOracleVirtualized(sources.DataSource):
+    """dhcp fails in init-local timeframe unless paravirtualized
+
+    dhclient was previously broken in a way that caused local timeframe to
+    raise an exception. With dhcpcd, this exception doesn't get raised.
+    This means that we need to manually exit unless the instance is a container
+    """
+
+    def _get_data(self):
+        if not util.is_container():
```

Comment:
This isn't the right check, but @a-dubs is going to take over this effort and write the check which instances fail, and return False for those instances.